### PR TITLE
Add GitHub-related CI stuff

### PR DESCRIPTION
**Pull request by tobolar**
Chages of files as compared to those used for MSL:
```
.CI/
  check_deprecated_line_color.py  . . untouched
  check_html.py . . . . . . . . . . . untouched
  compiler.sh . . . . . . . . . . . . deleted
  Jenkinsfile . . . . . . . . . . . . Library build stuff deleted/commented
  parse.mos . . . . . . . . . . . . . load VehicleInterfaces

.CI/checkHTMLDoc/
  checkLinks.py . . . . . . . . . . . untouched
  checkTags.py  . . . . . . . . . . . untouched
  tidyHTML.py . . . . . . . . . . . . untouched
  __init__.py . . . . . . . . . . . . untouched

.CI/classCoverage/
  CountClasses.mos  . . . . . . . . . load VehicleInterfaces
  CountClasses.py . . . . . . . . . . untouched

.CI\compilers
  Dockerfile  . . . . . . . . . . . . untouched

.CI/Test  . . . . . . . . . . . . . . deleted

.github/workflows
  checkCI.yml . . . . . . . . . . . . use for VehicleInterfaces; Library build stuff deleted/commented
  checkCron.yml . . . . . . . . . . . untouched
```

I'm especially not sure if set properly `.CI/classCoverage/CountClasses.mos` and `.CI/parse.mos` - regarding `setModelicaPath`, etc.

### CODE REVIEW

**Comment by tobolar** [comment]
It seems to me the suggested CI changes already applied on itself. :-0

**Comment by beutlich** [comment]
> Obviously, the suggested CI changes already apply on itself. :-0

Only the GitHub Actions part, but not the Jenkins part @sjoelund

**Comment by sjoelund** [comment]
I guess the entire Jenkinsfile could be removed. Do you want the CountClasses script? It could be added as github action if you do want it.

File: .github/workflows/checkCI.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+name: CI
+
+on: [pull_request]
```
Comment (beutlich) [inline .github/workflows/checkCI.yml:3]:
```suggestion
on: [push, pull_request]
```

Comment (tobolar) [reply .github/workflows/checkCI.yml:3]: Is direct pushing even possible? Maybe pushing is handled here different compared to MSL. We can do this as you suggest and disable if necessary.

**Comment by tobolar** [comment]
> Do you want the CountClasses script?

I have no clue. Probably not at the moment.

**Comment by sjoelund** [comment]
> > Do you want the CountClasses script?
> 
> I have no clue. Probably not at the moment.

Then remove the following:

* `.CI/classCoverage/`
* `.CI/parse.mos`
* `.CI/Jenkinsfile`
* `.CI/compilers`

**Comment by tobolar** [comment]
Seems to be fine now. Any more comment? @sjoelund @beutlich

**Review by sjoelund — APPROVED** [review]

**Comment by tobolar** [comment]
@sjoelund @beutlich Thanks a lot!

**Pull request merged**